UDF Example failed rapids-cmake-version must be defined

**Describe the bug**
example cmakefile is not compatible with latest rapids-cmake anymore

https://github.com/NVIDIA/spark-rapids-examples/blob/branch-25.06/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt

```
17:04:17  [INFO] Executing tasks
17:04:17  [INFO]     [mkdir] Created dir: /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-302/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build
17:04:18  [INFO]      [exec] -- Configuring incomplete, errors occurred!
17:04:18  [INFO]      [exec] CMake Error at /home/jenkins/agent/workspace/jenkins-examples-udf-examples-native-302/examples/UDF-Examples/RAPIDS-accelerated-UDFs/target/cpp-build/RAPIDS.cmake:24 (message):
17:04:18  [INFO]      [exec]   The CMake variable rapids-cmake-version must be defined in the format
17:04:18  [INFO]      [exec]   MAJOR.MINOR.
17:04:18  [INFO]      [exec] Call Stack (most recent call first):
17:04:18  [INFO]      [exec]   CMakeLists.txt:21 (include)
```

**Steps/Code to reproduce bug**
Please provide a list of steps or a code sample to reproduce the issue.
Avoid posting private or sensitive data.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment details (please complete the following information)**
 - Environment location: [Standalone, YARN, Kubernetes, Cloud(specify cloud provider)]
 - Spark configuration settings related to the issue

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
@@ -14,7 +14,11 @@
 # limitations under the License.
 #=============================================================================
 
+# Keep the same with https://github.com/NVIDIA/spark-rapids-jni/blob/main/src/branch-xx.xx/cpp/CMakeLists.txt
+# the branch-xx.xx in the above line is the branch version
 cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
+
+# set to the cuDF version
 set(rapids-cmake-version "25.06")
 
 file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-25.06/RAPIDS.cmake

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
@@ -14,7 +14,7 @@
 # limitations under the License.
 #=============================================================================
 
-# Keep the same with https://github.com/NVIDIA/spark-rapids-jni/blob/main/src/branch-xx.xx/cpp/CMakeLists.txt
+# Keep the same with https://github.com/rapidsai/rapids-cmake/blob/branch-xx.yy/RAPIDS.cmake
 # the branch-xx.xx in the above line is the branch version
 cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
 

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
@@ -14,7 +14,8 @@
 # limitations under the License.
 #=============================================================================
 
-cmake_minimum_required(VERSION 3.28.6 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
+set(rapids-cmake-version "25.06")
 
 file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-25.06/RAPIDS.cmake
      ${CMAKE_BINARY_DIR}/RAPIDS.cmake)

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
@@ -18,10 +18,10 @@
 # the branch-xx.xx in the above line is the branch version
 cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
 
-# set to the cuDF version
+# set to the rapids-cmake version
 set(rapids-cmake-version "25.06")
 
-file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-25.06/RAPIDS.cmake
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-${rapids-cmake-version}/RAPIDS.cmake
      ${CMAKE_BINARY_DIR}/RAPIDS.cmake)
 include(${CMAKE_BINARY_DIR}/RAPIDS.cmake)
 

